tests/Integration/TestCase.php

```diff
@@ -51,6 +56,26 @@ public function setUp($defaultSerializer = '')
                 'publisher' => 'Bloody Fantasy inc.',
             ],
         ];
+
+	    $this->testPosts = [
```

Comment:
Why have you added a second test set here?

I don't really like the name as these are books, not posts.